components/webapps/blacklist-whitelist-service/src/main/java/com/wso2telco/services/bw/Queries.java

```diff
@@ -436,88 +334,30 @@ public Response getAllApplicationsByUser(@PathParam("userId") String userId,@Pat
 
 	@GET
 	@Path("/apis/{userId}/{appId}")
-	@Produces("application/json")
-	public Response getAllApisByUserAndApp(@PathParam("userId") String userId,@PathParam("appId") String appId) throws SQLException, BusinessException {
-		//Gson gson = new GsonBuilder().serializeNulls().create();
-		if (userId!=null && appId!=null) {
-			String jsonString = blackListWhiteListService.getAllApisByUserAndApp(userId,appId);
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response getAllApisByUserAndApp(@PathParam("userId") String userId, @PathParam("appId") String appId) {
+		String jsonString = null;
```

Comment:
avoid declaring variable here. Instead, define it at the same line of initializing (line 344)

Reply:
updated